apps/ui/lev-help.md

```diff
@@ -0,0 +1,72 @@
+# Help with LEV
+
+If you spot a problem with any page or function of this website, please let us
+know.
```

Comment:
Having a link to the contact page would probably be a good idea/and or other avenues of contact

Reply:
Contact details are further down on the page. Have amended the paragraph to make this clearer.